src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -329,6 +329,15 @@ private function addProfilerSection(ArrayNodeDefinition $rootNode)
                         ->booleanNode('only_main_requests')->defaultFalse()->end()
                         ->scalarNode('dsn')->defaultValue('file:%kernel.cache_dir%/profiler')->end()
                         ->booleanNode('collect_serializer_data')->info('Enables the serializer data collector and profiler panel')->defaultFalse()->end()
+                        ->arrayNode('cli')
+                            ->canBeEnabled()
```

Comment:
can we imagine something like https://symfony.com/blog/new-in-symfony-5-4-profiler-improvements#conditional-profiling ?

when executing for example `bin/console app:command --profile` to mimic the http query param

Reply:
It was a future scope, but it is now the default to enable profiling instead of semantic configuration (which has been dropped altogether).